src/component/strategies/StrategiesList/StrategySwitch/StrategySwitch.tsx

```diff
@@ -0,0 +1,38 @@
+import { VFC } from 'react';
+import PermissionSwitch from 'component/common/PermissionSwitch/PermissionSwitch';
+import { UPDATE_STRATEGY } from 'component/providers/AccessProvider/permissions';
+import { Tooltip } from '@mui/material';
+
+interface IStrategySwitchProps {
+    deprecated: boolean;
+    onToggle: (state: boolean) => void;
+    disabled?: boolean;
+}
+
+export const StrategySwitch: VFC<IStrategySwitchProps> = ({
+    deprecated,
+    disabled,
+    onToggle,
+}) => {
+    const onClick = () => {
+        onToggle(deprecated);
+    };
+
+    return (
+        <Tooltip
+            title={
```

Comment:
Looks like this label gets applied to a span, and not to the input element. 

We should also make the tooltip work for disabled switches somehow.